modules/schedule/index.js

```diff
@@ -367,7 +367,7 @@ module.exports.commands = {
       if(Object.keys(schedules).length == 0)
         return reply("There are no schedules to delete.");
 
-      if(digest in schedules)
+      if(!(digest in schedules))
```

Comment:
Minor nits that the rest of the code has stylistic choices of using `{}` for one line bodies and I think that's generally more popular nowadays anyways, and `in` is *often* not what you want, though it should be fine here.